src/java.base/share/classes/java/util/jar/JarOutputStream.java

```diff
@@ -81,6 +81,10 @@ public JarOutputStream(OutputStream out) throws IOException {
      * The current time will be used if the entry has no set modification
      * time.
      *
+     * The compressed size field will be recalculated for compressed (i.e.
```

Comment:
I think the wording is better, but not sure we should state "compressed size field" as ZipEntry.setCompressedSize refers to "Sets the size of the compressed entry data."   and "the compressed size to set" 

I think I would omit "field" from the above so it reads  similar to "The compressed size will be..." or "The compressed size value will be..."